apps/betterangels-backend/notes/tests/test_queries.py

```diff
@@ -340,6 +378,68 @@ def test_notes_query_filter_by_teams(
             for idx, note_label in enumerate(returned_note_labels):
                 self.assertEqual(notes[idx]["id"], getattr(self, note_label)["id"])
 
+    @parametrize(
+        ("name_search, expected_results_count, returned_authors"),
+        [
+            ("Maximoff", 1, ["interaction_author_2"]),  # Two notes have "deets" in public details
+            ("Pietro Maximoff", 0, None),  # One note has "deets" in public details and "coop" in client name
+            (
+                "Alex",
+                2,
+                ["interaction_author", "interaction_author_3"],
+            ),  # One note has "more" in public details
+        ],
+    )
+    def test_interaction_authors_filter(
+        self,
+        name_search: Optional[str],
+        expected_results_count: int,
+        returned_authors: Optional[list[str]],
+    ) -> None:
+        self.graphql_client.force_login(self.org_1_case_manager_2)
+
+        query = """
+            query InteractionAuthors($filters: InteractionAuthorFilter) {
+                interactionAuthors(filters: $filters) {
+                    totalCount
+                    results {
+                        id
+                        firstName
+                        lastName
+                        middleName
+                    }
+                }
+            }
+        """
+
+        test_user_map = {
+            "interaction_author": baker.make(User, first_name="Alexa", last_name="Danvers", middle_name="J."),
+            "client": baker.make(User, first_name="Alex", last_name="Johnson"),
+            "interaction_author_2": baker.make(User, first_name="Wanda", last_name="Maximoff", middle_name="A."),
+            "interaction_author_3": baker.make(User, first_name="Alexandria", last_name="Daniels", middle_name="M."),
+        }
+
+        interaction_author = test_user_map["interaction_author"]
+        interaction_author_2 = test_user_map["interaction_author_2"]
+        interaction_author_3 = test_user_map["interaction_author_3"]
+
+        perm_group = permission_group_recipe.make()
+        perm_group.organization.add_user(interaction_author)
```

Comment:
you can do `perm_group.organization.users.add(interaction_author, interaction_author_2, interaction_author_3)